config/dev.exs

```diff
@@ -39,7 +39,7 @@ config :apr, :basic_auth, [
 config :phoenix, :stacktrace_depth, 20
 
 config :kafka_ex,
-  brokers: [{"192.168.99.100", 9092}],
-  consumer_group: System.get_env("KAFKA_CONSUMER_GROUP") || "kafka_ex",
-  disable_default_worker: true,
+  brokers: [{"ip-10-0-0-49.ec2.internal", 9092}, {"ip-10-0-0-248.ec2.internal", 9092}],
```

Comment:
We had problems parsing out a string in the form of "host:port,host:port".  We can always try and leave port hardcoded but pass a string in the form of "host,host"